components/whyd/2022/Terminal.vue

```diff
@@ -25,6 +25,11 @@
             :height="text?.height ?? '100px'"
             class="terminal-image"
           />
+          <Whyd2022ChatText
+            v-if="text.lastLineInCommand"
+            :key="`last${text.id}`"
+            content="{End of Command | underline block}"
+          />
```

Comment:
This is added to the template itself instead of terminal lines because it was appearing above images.